tests/test_cases/test_cocotb/test_cocotb.py

```diff
@@ -1015,7 +1015,8 @@ def raise_soon():
       File ".*test_cocotb\.py", line \d+, in raise_soon
         yield cocotb\.triggers\.First\(raise_inner\(\)\)
       File ".*triggers\.py", line \d+, in _wait
-        return \(yield first_trigger\)[^\n]*
+        return \(await first_trigger\)[^\n]*
+    .*
```

Comment:
I feel like there is a better way, but I don't know it.

Reply:
What does this traceback look like now?

`[\s\S]*` would also work here, and avoid the need for "dot all"

Reply:
There are potentially many lines there chaining through `__await__`s there due to `First` using `await` on `raise_inner()`.
```
Traceback (most recent call last):
  File "/home/kaleb/dev/cocotb/tests/test_cases/test_cocotb/test_cocotb.py", line 922, in _check_traceback
    yield running_coro
  File "/home/kaleb/dev/cocotb/tests/test_cases/test_cocotb/test_cocotb.py", line 1007, in raise_soon
    yield cocotb.triggers.First(raise_inner())
  File "/home/kaleb/dev/cocotb/cocotb/triggers.py", line 745, in _wait
    return await first_trigger  # the first of multiple triggers that fired
  File "/home/kaleb/dev/cocotb/cocotb/triggers.py", line 128, in __await__
    return (yield self)
  File "/home/kaleb/dev/cocotb/cocotb/decorators.py", line 204, in __await__
    return (yield self)
  File "/home/kaleb/dev/cocotb/tests/test_cases/test_cocotb/test_cocotb.py", line 1002, in raise_inner
    raise ValueError('It is soon now')
ValueError: It is soon now
```

Reply:
I'd prefer you list those explicitly, because it's ugliness in the traceback we may want to try to remove in future.
```suggestion
      File ".*triggers.py", line \d+, in __await__
        return \(yield self\)
      File ".*decorators.py", line \d+, in __await__
        return \(yield self\)
```